### ISSUE TITLE
How to sanitize messages?

### ISSUE THREAD

**Issue by szczynk** [dword-design/nuxt-mail#76]
How exactly sanitize messages in this module?

**Comment by dword-design** [comment]
@szczynk Good point. We did some work on security already in this module regarding the email addresses, but sanitization is probably a thing we should enable too. I think it could be done with a change I proposed in another issue. It would make the message generation dynamic on server side and there you could add some more logic.

Have a look at this comment where I describe it:
https://github.com/dword-design/nuxt-mail/issues/68#issuecomment-883610093

Would be great to get your feedback on this!

**Comment by dword-design** [comment]
@szczynk I have an [open PR](https://github.com/dword-design/nuxt-mail/pull/112) that would solve the issue feel free to have a look!
